src/stores/Frame/FrameStore.ts

```diff
@@ -2887,6 +2887,7 @@ export class FrameStore {
             smoothingFactor: config.pixelAveragingEnabled ? config.pixelAveraging : 1,
             fractional: config.fractionalIntensity,
             threshold: config.thresholdEnabled ? config.threshold : NaN,
+            thresholdOption: config.thresholdOption,
```

Comment:
Just wondering if there is no Stokes I and Plinear (i.e. the dropdown selection is not shown), or if the threshold is not enabled, would it be unnecessary to send either stokes I or computed PI to the backend?